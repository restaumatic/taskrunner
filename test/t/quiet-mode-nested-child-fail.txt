# quiet
echo "Toplevel output before nested call"
taskrunner -n nested sh -c 'echo "Nested output before failure"; exit 1' || echo "Handling nested failure"
echo "Toplevel continues after nested failure"