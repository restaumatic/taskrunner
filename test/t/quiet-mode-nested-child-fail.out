-- output:
[nested] stdout | Nested output before failure
